app/scripts/components/searchdetails.js

```diff
@@ -47,14 +50,16 @@ var exports = (function () {
       };
 
       // get search templates
-      Promise.all([fetchUsers(), fetchSearchTemplate()])
+      Promise.all([fetchUsers(), fetchSearchTemplate(), fetchVersions()])
         .then((jsons) => {
           var users = jsons[0];
           var template = jsons[1];
+          var versions = jsons[2];
```

Comment:
You could use [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment). I believe our code is transpiled by babel, so it should be compatible. `[users, template, versions] = jsons;`